src/ca/wolfram/beta/symath/ConstantNode.java

```diff
@@ -0,0 +1,44 @@
+package ca.wolfram.beta.symath;
+
+import java.util.List;
+
+/**
+ * Created by Allan Wang on 2017-05-06.
+ */
+public class ConstantNode implements MathNode {
+
+    private final double constant;
+
+    private ConstantNode(double constant) {
+        this.constant = constant;
+    }
+
+    public static ConstantNode create(double constant) {
+        return new ConstantNode(constant);
+    }
+
+    @Override
+    public NodeType getType() {
+        return NodeType.CONSTANT;
+    }
+
+    @Override
+    public List<MathNode> getChildren() {
+        return null;
```

Comment:
would it be worth it to throw a RuntimeException? otherwise it might mean someone is not checking for null